### ISSUE TITLE
Can Maximize be aware of notification center?

### ISSUE THREAD

**Issue by jmontana** [fikovnik/ShiftIt#272]
I would love to be able to maximize the window with a hot key, but have it leave room on the left for the notifications. Sometimes I have alerts that need to remain on screen (meeting in 10 minutes, a hot Reminder, etc.) but I need to be able to see beneath them for the current window. Right now, I do a Cmd-Ctrl-Opt-Left arrow, followed by a few Cmd-Ctrl-Opt-+ until the window is as close to the left edge of the notification as I can, but it's a lot of keystrokes, and it's not precise.

**Comment by fikovnik** [comment]
I always though that the notification are sticky and top front - that is they stay up. I remember the calendar items poping up - meetings in 5 minutes .... - and staying there unless I clicked on them.

**Comment by jmontana** [comment]
You're right, they do. The problem is sometimes I need to interact with a window that's beneath the notification. Rather than moving the window, interacting, and moving the window back, I'd like a hot key that would automatically resize the window to account for the notifications.

**Comment by fikovnik** [comment]
I see. It would be great to have a support for custom actions, but they are not there yet. Do you want to give it a try?

**Comment by jmontana** [comment]
I'd love to, but don't know that I have the time. Maybe I'll pull the code and play around with it and see what I can come up with.
